### ISSUE TITLE
Add widget for Webform selection

### ISSUE THREAD

**Issue by alliomeria** [esmero/webform_strawberryfield#77]
# What?
Instead of only allowing a single Webform to be accessed per Form Mode, add a "widget" that could be used to select/pair a particular Webform based on a JSON "type" key (e.g. Article).

# Why?
To allow more fine-tuned configuration for what Webforms are being used for editing different types of objects, and/or are being accessed by different types of Users.

**Comment by DiegoPino** [comment]
@alliomeria we can reuse this one with a different title for the "File tagging" webform element if you want.

**Comment by alliomeria** [comment]
@DiegoPino Good idea! Sorry about the duplicative issue. Reopened this and amended with basic information for the "file tagging" issue.
